codeforamerica/vita-min
# FYST-1340 state specific copy on post submit screens

## Link to pivotal/JIRA issue
- https://codeforamerica.atlassian.net/browse/FYST-1340
## Is PM acceptance required? (delete one)
- Yes - don't merge until JIRA issue is accepted!
- No - merge after code review approval

**Reminder**: merge main into this branch and get green tests before merging to main
## What was done?
- A bunch of content updates to the return status page
## How to test?
- I started to write some controller specs to check that the right content shows up but it was starting to feel little time consuming to cover all cases that way and I'm not sure what our philosophy on testing for content is? Do we believe that tests should check all the copy or is it enough to have acceptance testers just look at it?
## Screenshots (for visual changes)
there's a little variation between states but not enough to get screenshots of all of them

pending:
![Screenshot 2024-12-06 at 11 35 27 AM](https://github.com/user-attachments/assets/35aa1511-f96f-43fc-b67f-092973f9db64)

accepted (owe)
![Screenshot 2024-12-03 at 2 24 07 PM](https://github.com/user-attachments/assets/8ea06dc4-928e-42c2-b026-21a71f82c281)

accepted (refund)
![Screenshot 2024-12-03 at 11 31 28 AM](https://github.com/user-attachments/assets/57fb1594-ace3-46ad-a66d-66c337d5ea6e)

rejected
![Screenshot 2024-12-06 at 11 40 22 AM](https://github.com/user-attachments/assets/c17baae7-2384-4a00-a883-74dc657cad8c)
